db/config.cc

```diff
@@ -1135,6 +1135,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , replication_strategy_warn_list(this, "replication_strategy_warn_list", liveness::LiveUpdate, value_status::Used, {locator::replication_strategy_type::simple}, "Controls which replication strategies to warn about when creating/altering a keyspace. Doesn't affect the pre-existing keyspaces.")
     , replication_strategy_fail_list(this, "replication_strategy_fail_list", liveness::LiveUpdate, value_status::Used, {}, "Controls which replication strategies are disallowed to be used when creating/altering a keyspace. Doesn't affect the pre-existing keyspaces.")
     , service_levels_interval(this, "service_levels_interval_ms", liveness::LiveUpdate, value_status::Used, 10000, "Controls how often service levels module polls configuration table")
+    , materialized_views_rmw_admission(this, "materialized_views_rmw_admission", liveness::LiveUpdate, value_status::Used, true, "Controls if materialized views reads in rmw use admission control")
+    , materialized_views_rmw_admission_fraction(this, "materialized_views_rmw_admission_fraction", liveness::MustRestart, value_status::Used, 1.0, "The fraction/multiplier for the materialized views concurrency relative to the read concurrency")
```

Comment:
We can easily make this LiveUpdate too, we already have experience in changing memory amount of semaphores on-the-fly, with the SL semaphroes.